examples/acquiring_data/hmi.py

```diff
@@ -27,26 +27,28 @@
 
 ###############################################################################
 # Now we can see what results we obtained from our search.
+# Notice we have two files. One is the full disk image we plan to display
+# and the other is a synoptic version of said image.
+
 print(result)
 
 ###############################################################################
 # Once we are happy with the results obtained from the search.
 # We can download the data with `Fido.fetch` .
 # In this case we only want one file so we can index the result.
 
-# Notice we have two files. One is the full disk image we plan to display
-# and the other is a synoptic version of said image.
-downloaded_files = Fido.fetch(result)
+downloaded_file = Fido.fetch(result[0, 0])
```

Comment:
Given the slightly peculiar semantics of this slicing I suggest we add a comment like
```
# Slice the first record returned by the first client
```
or similar to explain both the indices.

Reply:
Good point

Reply:
Added a sentence abov e